runtime-testsuite/resources/org/antlr/v4/test/runtime/templates/CompositeParsers/DelegatorInvokesDelegateRuleWithArgs.stg

```diff
@@ -28,5 +28,5 @@ WS : (' '|'\n') -> skip ;
 
 slaveGrammar(grammarName) ::= <<
 parser grammar S;
-a[int x] returns [int y] : B {<write("\"S.a\"")> $y=1000;} ;
```

Comment:
if it works out of the bow, why not, but I will not modify the Python templates to support this
managing indents in Python is notoriously hazardous

> Le 21 juin 2015 à 04:06, Sam Harwell <notifications@github.com> a écrit :
> 
> In runtime-testsuite/resources/org/antlr/v4/test/runtime/templates/CompositeParsers/DelegatorInvokesDelegateRuleWithArgs.stg <https://github.com/antlr/antlr4/pull/927#discussion_r32887534>:
> 
> > @@ -28,5 +28,5 @@ WS : (' '|'\n') -> skip ;
> >  
> >  slaveGrammar(grammarName) ::= <<
> >  parser grammar S;
> > -a[int x] returns [int y] : B {<write("\"S.a\"")> $y=1000;} ;
> @ericvergnaud <https://github.com/ericvergnaud> Notice that my proposed form is two consecutive, independent actions. I would expect the Python target to emit them each on their own line, and thus not need the separating ;.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/antlr/antlr4/pull/927/files#r32887534>.
>